desc/objectives/_neoclassical.py

```diff
@@ -283,3 +283,120 @@ def compute(self, params, constants=None):
             **self._hyperparam,
         )
         return constants["transforms"]["grid"].compress(data["effective ripple"])
+
+
+class OldEffectiveRipple(EffectiveRipple):  # noqa: D101
```

Comment:
maybe keep it a single class but add a `interpolation_method` flag to be `1D` or `2D`

Reply:
that would be clean but we did not know if combining the objectives would have disadvantage for jit compilation